newrelic.yml

```diff
@@ -452,6 +452,11 @@ common: &default_settings
 # rake.connect_timeout: 10
 
 # Specify an array of Rake tasks to automatically instrument.
+# This configuration option converts the Array to a RegEx list.
+# If you'd like to allow all tasks by default, use [.+].
```

Comment:
It's hard to correct for my bias of knowing how this works to decide what others would benefit from seeing. Maybe the use of quotes or the inclusion of `rake.tasks` would be help, but I'm not sure. Here are all 3 side by side for comparison:

- `, use [.+].`
- `, use "[.+]".`
- `, use "rake.tasks: [.+]"`

Reply:
Excellent point. I think my favorite is a riff on the last: 
```suggestion
# If you'd like to allow all tasks by default, use `rake.tasks: [.+]`.
```

What do you think about back tics vs. quotes?

Reply:
Good stuff! 👍